aws_health_to_slack/program.py

```diff
@@ -87,13 +87,18 @@ def _affected_aws_entities(events: list[dict]) -> list[dict]:
     try:
         aws = boto3_client("aws_connection", "health")
         arns = [event["arn"] for event in events]
+
+        if not arns:
+            return []
```

Comment:
I think a clearer place for this check is after line 26 (which is also affected by line 25, but won't fail if `events` is empty.

The hard-to-understand subtlety about this check here is that there's no risk that a (real) event won't contain an "arn" field, but it is possible that `events` is empty, in which case `filter` will not contain a required field (`eventArns`).